Signal/src/ViewControllers/SignalAttachment.swift

```diff
@@ -266,10 +275,26 @@ class SignalAttachment: NSObject {
         return SignalAttachment.audioUTISet.contains(dataUTI)
     }
 
+    public var isText: Bool {
+        return SignalAttachment.textUTISet.contains(dataUTI)
+    }
+
     public class func pasteboardHasPossibleAttachment() -> Bool {
         return UIPasteboard.general.numberOfItems > 0
     }
 
+    public class func pasteBoardHasText() -> Bool {
```

Comment:
nit: *seems* like we've settled on Pasteboard as one word, we should be consistent.